frog/imports/ui/Admin.jsx

```diff
@@ -1,17 +1,132 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { createContainer } from 'meteor/react-meteor-data';
 
 import { Logs } from '../api/logs';
 import { Sessions, importSession } from '../api/sessions';
 import {
   Activities,
   Operators,
+  Connections,
   importActivity,
   importOperator,
+  importConnection,
   deleteDatabase
 } from '../api/activities';
 import { Graphs, importGraph } from '../api/graphs';
 
+const fakeDatabase = {
```

Comment:
can you put this in a separate file please? eventually we might want to have several saved graphs etc.